oam/install_scripts/post-install

```diff
@@ -190,22 +216,16 @@ if [ $user = "root" ]; then
     touch $RCFILE    
     chmod +x $RCFILE
```

Comment:
Not part of this patch, but this is bad.  Don't change perms of system init scripts.